src/components/toggle/_toggle.scss

```diff
@@ -211,3 +217,178 @@
     }
   }
 }
+
+@mixin toggle--experimental {
+  .#{$prefix}--toggle {
+    @include hidden;
+
+    &:focus {
+      outline: none;
+    }
+  }
+
+  .#{$prefix}--toggle__label {
+    @include font-family;
+    position: relative;
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    margin: $spacing-xs 0;
+  }
+
+  .#{$prefix}--toggle__appearance {
+    position: relative;
+    width: rem(48px);
+    height: rem(24px);
+
+    // Toggle background oval
+    &:before {
+      position: absolute;
+      display: block;
+      content: '';
+      background-color: $ui-03;
+      cursor: pointer;
+      box-sizing: border-box;
+      height: rem(24px);
+      width: rem(48px);
+      border-radius: 0.9375rem;
+      border: 1px solid $ui-04;
+      top: 0;
+    }
+
+    // Toggle circle
+    &:after {
+      box-sizing: border-box;
+      position: absolute;
+      display: block;
+      cursor: pointer;
+      left: 3px;
+      top: 3px;
+      width: rem(18px);
+      height: rem(18px);
+      background-color: $ui-02;
+      border-radius: 50%;
+      content: '';
+      transition: $transition--base $carbon--standard-easing;
+    }
+  }
+
+  .#{$prefix}--toggle--small + .#{$prefix}--toggle__label .#{$prefix}--toggle__appearance {
+    width: rem(32px);
+    height: rem(16px);
+
+    &:before {
+      box-sizing: border-box;
+      height: rem(16px);
+      width: rem(32px);
+      border-radius: 0.9375rem;
```

Comment:
What's the reasoning behind this value? Is it 15px? Maybe we should just use `rem(15px)`, as it seems a bit easier to understand

Reply:
👍 done